packages/styles/select.css

```diff
@@ -83,3 +83,9 @@
 .Field__select--wrapper.Field--has-error select:focus:hover {
   border-color: var(--field-border-color-error-hover);
 }
+
+@media (max-width: 23rem) {
```

Comment:
I don't think this is the change that we need.

What if the component is inside of a smaller container on a larger viewport? It could still overflow. I believe the desired behavior should be we need to change `min-width` to `width` and then set a `max-width: 100%` so the select element can responsively flex.

Reply:
That works!